dataframe.py

```diff
@@ -62,19 +58,16 @@ def load_dataframe():
 
     return dataframe
 
-
+# train test
 def train_test(dataframe):
-    '''
-    '''
     X = list(dataframe["image"])
     y = list(dataframe["target"])
 
     return train_test_split(X, y, train_size=0.40, random_state=13)
 
 
+# pca model
```

Comment:
pca model is not used in our code, right ? remove it

Reply:
it is used in training_model.py sir